source/drivers/swift.txt

```diff
@@ -0,0 +1,81 @@
+.. _swift-language-center:
+
+.. include:: /includes/unicode-checkmark.rst
+
+====================
+MongoDB Swift Driver
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+This is the official MongoDB Swift Driver.
+
+.. admonition:: Alpha Release
+   :class: important
+
+   The MongoDB Swift Driver is currently in alpha.
+
+- `General documentation for the MongoDB Swift Driver <https://mongodb.github.io/mongo-swift-driver>`__
+
+- `Get Started with Swift <https://mongodb.github.io/mongo-swift-driver/index.html#example-usage>`__
+
+- `Changelog <https://github.com/mongodb/mongo-swift-driver/releases>`__
+
+- `Source Code <https://github.com/mongodb/mongo-swift-driver/>`__
+
+
+Installation
+------------
+
+See `Installation <https://mongodb.github.io/mongo-swift-driver/index.html#installation>`__
+
+
+Connect to MongoDB Atlas
+------------------------
+
+To connect to a :atlas:`MongoDB Atlas </>` cluster, use the :atlas:`Atlas connection string </driver-connection>` for your cluster:
+
+.. code-block:: swift
+
+   import MongoSwift
+
+   let client = try MongoClient("mongodb+srv://<username>:<password>@<cluster-url>/test?retryWrites=true&w=majority")
+   let db = client.db("test")
+
+   // your application logic
+   // free all resources
+   cleanupMongoSwift()
+
+Compatibility
+-------------
+
+MongoDB Compatibility
+~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/mongodb-compatibility-table-swift.rst
+
+Language Compatibility
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/language-compatibility-table-swift.rst
+
+How to get help
+---------------
+
+- Join our `Google Group <https://groups.google.com/forum/#!forum/mongodb-go-driver>`__.
```

Comment:
Are these links (line 74-75) correct for the Swift driver? They appear to link to the go driver forums.